pkg/router/template/plugin.go

```diff
@@ -99,7 +99,7 @@ func NewTemplatePlugin(cfg TemplatePluginConfig) (*TemplatePlugin, error) {
 	globalFuncs := template.FuncMap{
```

Comment:
We need to doc what these do... can we at least put a one-line comment on them to summarize usage?